docs/programming-languages.md

```diff
@@ -17,6 +17,10 @@ Python is a high-level, general-purpose, interpreted scripting language. It is
 great for beginners, while also powerful for experienced coders.If you are new
 to programming and unsure which language to choose, Python is a great choice!
 
+We feel that Python is the best option if you are just starting programming.
+The [ev3python.com](http://ev3python.com) by @ndward is an excellent guide
```

Comment:
Oh, I read that... I just didn't realize that this sentence was different 😪 I like it how I said I shouldn't code review at 5:00 in the morning yesterday so instead I reviewed this at 6:00.

Anyway, yes I like that one better :laughing: Although, the capitalization that we use (based on what is used upstream) is all-caps LEGO MINDSTORMS EV3. Sorry for making this so difficult.